nncf/experimental/torch/fx/model_transformer.py

```diff
@@ -29,7 +32,7 @@ class FXModelTransformer(ModelTransformer):
     """
 
     def __init__(self, model: torch.fx.GraphModule):
-        super().__init__(model)
+        super().__init__(deepcopy(model))
```

Comment:
For example, model copying was done on the algorithm's side -  https://github.com/openvinotoolkit/nncf/blob/develop/nncf/quantization/algorithms/bias_correction/algorithm.py#L141.
Also, do we need to have a copy for all transformations or only for extraction? Thus, you can make copying only for this case and do not overload other pipelines.

Reply:
> That's because all of the transformations in torch fx except model extraction are made inplace. Since model transformer is expected to not modify input input model, I've added this deep copy

Why is MT expected not to modify the input model?
For PT backend model transformer works without deepcopy.

Reply:
Let's discuss it together and choose a way to cope with the model copying on the tomorrow sink

Reply:
Deepcopy is removed, a comment is added. @AlexanderDokuchaev, please check